stages/org.osbuild.grub2

```diff
@@ -141,6 +148,10 @@ SCHEMA = """
       }
     }
   },
+  "saved_entry": {
+    "description": "Set the variable of the same name in `grubenv`.",
```

Comment:
What format should this have, and how would we discover it in composer (we have NEVRA of the kernel package).

Reply:
Should be called exactly like the corresponding loader entry, i.e. "ffffffffffffffffffffffffffffffff-5.6.6-300.fc32.x86_64", which is `${machine-id}-${kernel-version}`. The former is always `ff...` for us, the latter needs to be figured out in composer after dep-solving. I can add some more help text, maybe in the doc text above.

Reply:
Should we make `kernel_version` the parameter, as the `fffffff...` is an osbuild thing?

Reply:
After discussing offline: the format might change in the future, better not hardcode the `fffff`s.